independent-publisher-connectors/SOS Inventory/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2218 @@
+							"example": "8"
+						},
+						"name": {
+							"type": "string",
+							"description": "Sales Order Sales Rep Name",
+							"example": "Ricky Roma"
+						}
+					}
+				},
+				"channel": {
+					"type": "string",
+					"description": "Sales Order Channel"
+				},
+				"department": {
+					"type": "object",
+					"description": "Sales Order Department",
+					"properties": {
+						"id": {
+							"type": "number",
+							"description": "Sales Order Department ID",
+							"example": "1"
+						},
+						"name": {
+							"type": "string",
+							"description": "Sales Order Department Name",
+							"example": "Retail"
+						}
+					}
+				},
+				"priority": {
+					"type": "string"
+				},
+				"assignedToUser": {
+					"type": "string"
+				},
+				"orderStage": {
+					"type": "string"
+				},
+				"taxCode": {
+					"type": "string"
+				},
+				"currency": {
+					"type": "string"
+				},
+				"serial": {
+					"type": "string"
+				},
+				"linkedTransaction": {
+					"type": "string"
+				},
+				"linkedInvoices": {
+					"type": "array",
+					"items": {}
+				},
+				"linkedShipments": {
+					"type": "array",
+					"items": {}
+				},
+				"transactionLocationQuickBooks": {
+					"type": "string"
+				},
+				"exchangeRate": {
+					"type": "number",
+					"example": "1"
+				},
+				"customerMessage": {
+					"type": "string"
+				},
+				"statusMessage": {
+					"type": "string"
+				},
+				"comment": {
+					"type": "string"
+				},
+				"customerNotes": {
+					"type": "string"
+				},
+				"customFields": {
+					"type": "string"
+				},
+				"customerPO": {
+					"type": "string",
+					"example": "416453007"
+				},
+				"depositPercent": {
+					"type": "number",
+					"example": "0"
+				},
+				"depositAmount": {
+					"type": "number",
+					"example": "0"
+				},
+				"subTotal": {
+					"type": "number",
+					"example": "1380"
+				},
+				"discountPercent": {
+					"type": "number",
+					"example": "0"
+				},
+				"discountAmount": {
+					"type": "number",
+					"example": "0"
+				},
+				"taxPercent": {
+					"type": "number",
+					"example": "0"
+				},
+				"taxAmount": {
+					"type": "number",
+					"example": "0"
+				},
+				"shippingAmount": {
+					"type": "number",
+					"example": "0"
+				},
+				"total": {
+					"type": "number",
+					"example": "1380"
+				},
+				"discountTaxable": {
+					"type": "boolean"
+				},
+				"shippingTaxable": {
+					"type": "boolean"
+				},
+				"dropShip": {
+					"type": "boolean"
+				},
+				"closed": {
+					"type": "boolean"
+				},
+				"archived": {
+					"type": "boolean"
+				},
+				"summaryOnly": {
+					"type": "boolean"
+				},
+				"hasSignature": {
+					"type": "boolean"
+				},
+				"earliestDueDate": {
+					"type": "string",
+					"format": "date-time",
+					"example": "2022-02-11T00:00:00"
+				},
+				"statusLink": {
+					"type": "string",
+					"example": "https://.orderfacts.com/Go?o=2ydI%2fS%2bS7n%2bUds9ac%2fYSHg%3d%3d"
```

Comment:
@Harold-Anderson can you please replace the value for this example to some string? E.g. "A link to OrderFacts website"